docs/website/root/manual/getting-started/run-signer-node.md

```diff
@@ -113,6 +113,30 @@ Note that this guide works on a Linux machine only.
 
 :::
 
+:::info
+
+Compare the version of your Cardano node with the minimum supported versions listed in the [`cardano-min-versions.json`](https://github.com/input-output-hk/mithril/blob/main/cardano-min-versions.json) to verify its compatibility with the Mithril signer.
```

Comment:
This should be updated with `networks.json`

Reply:
Done ✅